pybind_interface/pybind_main.h

```diff
@@ -216,7 +216,10 @@ PYBIND11_MODULE(qsim, m) {
     .def_readwrite("num_qubits", &Circuit::num_qubits)
     .def_readwrite("gates", &Circuit::gates);
 
-  py::bind_vector<NoisyCircuit>(m, "NoisyCircuit");
+  py::class_<NoisyCircuit>(m, "NoisyCircuit")
```

Comment:
This should allow us to remove the `PYBIND_MAKE_OPAQUE` line at the top of the file:
https://github.com/quantumlib/qsim/blob/6ad4cd5a3f34cc31012987b6f77ce3a564206f48/pybind_interface/pybind_main.h#L32

Reply:
Done.